CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Added
+
+- Added the `updateTrustedDomain` GraphQL API to modify the list of trusted
+  domains.
```

Comment:
Could you clarify the meaning of `modify the list` here please? My understanding of `updateTrustedDomain` GraphQL API is that the API allows updating a single trusted domain. So I think `... to modify a trusted domain.` is rather straightforward. But it could be that you intended something else, so please let me know what you think.